pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -313,3 +313,14 @@ def test_order_aggregate_multiple_funcs():
     expected = pd.Index(['sum', 'max', 'mean', 'ohlc', 'min'])
 
     tm.assert_index_equal(result, expected)
+
+
+def test_uint64_type_handling():
```

Comment:
can you parameterize on first, last. Does this also deal with min, max? if so pls add parameterizeation as well. Anything else?